style.css

```diff
@@ -0,0 +1,282 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+}
+body {
+  background-color: lightskyblue;
+
```

Comment:
please remove the space in your css file.

Reply:
mujeeb bhai its a good practice to give some space between two unsimilar properties. @mujeeb100